contrib/lib/src/space_helmet/helmet.rs

```diff
@@ -0,0 +1,131 @@
+use std::sync::atomic::{AtomicBool, Ordering};
+
+extern crate rocket;
+use rocket::fairing::{Fairing, Info, Kind};
+use rocket::http::Header;
+use rocket::Rocket;
+use rocket::{Request, Response};
+
+use space_helmet::policy::*;
+
+pub struct Helmet<'a> {
+    expect_ct_policy: Option<ExpectCTPolicy<'a>>,
+    no_sniff_policy: Option<NoSniffPolicy>,
+    xss_protect_policy: Option<XSSPolicy<'a>>,
+    frameguard_policy: Option<FramePolicy<'a>>,
+    hsts_policy: Option<HSTSPolicy>,
+    force_hsts_policy: Option<HSTSPolicy>,
+    force_hsts: AtomicBool,
+    referrer_policy: Option<ReferrerPolicy>,
+}
+
+//helper for Helmet.apply
```

Comment:
Comments, but regular `//` and docstring `///` `//!`, should be of the form:

```
// Regular comment.
/// Docstring.
//! Docstring.
```

That is, a space after the last `/` or `!` and a "sentence" thereafter.